src/frontend/parallelization_phase.nit

```diff
@@ -95,11 +117,24 @@ end
 		assert nullreturn isa AExpr
 		mainfun.n_block.as(ABlockExpr).n_expr.add(nullreturn)
 
+
 		# Create new body for the annotated fun
-		var s_newbody ="""
+		var s_newbody : nullable String = null
+		if nmethdef.n_signature.n_params.length != 0 then
```

Comment:
In general you should use the `non_empty` method here, it can be faster than using lenght. In this case it is not critical thought.